source/deployment/sso-saml-adfs.rst

```diff
@@ -151,17 +151,21 @@ Next, we export the identity provider certificate, which will be later uploaded
 
 You’re now about to finish configuring SAML for Mattermost!
 
-Configure SAML sign-in for Mattermost
+Configure SAML Sign-in for Mattermost
 --------------------------------------
 
 1. Start Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**.
-  - *SAML SSO URL*: **SAML 2.0/W-Federation URL** ADFS Endpoint you copied earlier.
-  - *Identity Provider Issuer URL*: ``Relying party trust identifier`` from ADFS you specified earlier.
-  - *Identity Provider Public Certificate*: ``X.509 Public Certificate`` you downloaded earlier.
+  - **SAML SSO URL:** **SAML 2.0/W-Federation URL** ADFS Endpoint you copied earlier.
+  - **Identity Provider Issuer URL:** ``Relying party trust identifier`` from ADFS you specified earlier.
+  - **Identity Provider Public Certificate:** ``X.509 Public Certificate`` you downloaded earlier.
 
 	.. image:: ../../source/images/adfs_22_mattermost_basics.PNG
+	
+Alternatively, you can create a metadata URL by appending "FederationMetadata/2007-06/FederationMetadata.xml" to the root URL of the ADFS server, for example: ``https://<adfs.domain.com>/federationmetadata/2007-06/FederationMetadata.xml>``. 
 
-2. Configure Mattermost to verify the signature. The *Service Provider Login URL* is the *SAML 2.0 SSO service URL* you specified in ADFS earlier.
+Then paste this URL in the **Identity Provider Metadata URL** field, then select **Get SAML Metadata from IdP**.
```

Comment:
Maybe add another phrase that gives the rationale for entering the metadata Url and clicking the button - something along the lines of: "clicking this button populates the SAML SSO URL and Identity Provider Issue URL and downloads the Idp certificate from the ADFS server - to save you from populating these fields manually".

Reply:
Thanks @catalintomai - I closed this PR and opened a new one (https://github.com/mattermost/docs/pull/3341) so I'll add this comment to the PR for reference.